src/SelectMulti.vue

```diff
@@ -332,9 +332,11 @@
 
 <template>
 	<div class="vue-accessible-select-multi" :class="{ disabled, open }">
-		<label :id="htmlId" class="combo-label" :class="{ 'sr-only': !labelIsVisible }">
-			{{ label }}
-		</label>
+		<slot name="custom-label">
```

Comment:
Could you please name it just `label` and add the same slot into SelectSingle so we have consistency.
Thanks